packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -1166,35 +1168,59 @@ export const PersonalDetailsForm = ({
+                                                                setFieldTouched(
+                                                                    'request_professional_status',
+                                                                    true,
+                                                                    true
+                                                                );
+                                                            }}
+                                                            label={localize(
+                                                                'I would like to be treated as a professional client.'
+                                                            )}
+                                                            id='request_professional_status'
+                                                            defaultChecked={!!values.request_professional_status}
+                                                            disabled={
+                                                                is_virtual ||
+                                                                !!form_initial_values.request_professional_status
+                                                            }
+                                                            greyDisabled
+                                                            className={classNames({
+                                                                'dc-checkbox-blue': is_appstore,
+                                                            })}
+                                                        />
+                                                    ) : (
+                                                        <div className='account-form__authentication-message'>
+                                                            <Icon height={20} width={30} icon='IcInfoBlue' />
+                                                            <Text as='p' size='xs'>
+                                                                <Localize
+                                                                    i18n_default_text='You’ll need to authenticate your account before requesting to become a professional client. <0>Authenticate my account</0>'
+                                                                    components={[
+                                                                        <a
+                                                                            key={0}
+                                                                            className='link link--red'
+                                                                            rel='noopener noreferrer'
+                                                                            target='_blank'
+                                                                            href={'/account/proof-of-address'}
```

Comment:
just a clarification -ehich page user should be redirected? proof of identity or address?

Reply:
we have InlineNoteWithIcon component, already written to reuse it. Could you please check if we can use it with those background color?
I believe we should create a lot of similar components

Reply:
it is used already in api token form

Reply:
@amina-deriv user will be redirect to the page where they haven't verified. Will add the logic here 🙏